src/init/consented/prepare-permutive.ts

```diff
@@ -157,7 +157,7 @@ const runPermutive = (
 			page: payload,
 		});
 	} catch (err) {
-		logger(err, { feature: 'commercial' }, false);
+		logger(err, 'commercial');
```

Comment:
It looks like this is actually `reportError` but passed as a parameter called `logger`.

While we're here can we keep the name `reportError` so it is less confusing.